geometry_msgs/Pose desired_pose
---
sensor_msgs/JointState[] joints